src/main/webapp/Overdose.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+
+<head>
+	<title>
+		Are They Overdosing?
+	</title>
+	<link rel="stylesheet" href="style_emergancy.css">
+	<div class="head-style">
+		FUTURE STYLE
+	</div>
+</head>
+
+<body>
+	<!-- Drop down emergancy menu -->
+	<div class="dropdown">
+		<button class="dropbtn"><b>EMERGENCY</b></button>
+		<div class="dropdown-content">
+		<a href="Overdose.html">Overdose</a>
+		<a href="Addiction.html">Addiction</a>
+		<a href="Resources.html">Resources</a>
+	</div>
+	
+	<!-- Overdose Content -->
+	<div class="heading">
+		<b>Overdoses</b>
+	</div>
+	
+	<div class="sub-heading">
+		<i><b>What are they and why do they happen?</b></i>
```

Comment:
This is all great information. I would probably put "Drug Overdoses and How to Deal with Them" at the top of an emergency page. There could be something like a question and answer walkthrough based on what the person reading knows. Did they just find a person in distress? Do they know or suspect what the person took? That could lead to how they can deal with it including contacting emergency services.